### ISSUE TITLE
Factory Default Settings

### ISSUE THREAD

**Issue by lucas-04** [Bionus/imgbrd-grabber#2645]
An extra button on each different configuration screen that resets the options on that screen to the factory settings

**Comment by Bionus** [comment]
Indeed, it's something I'd love to do (also one per field, like some apps have), but currently the way the options window is done it's not easily do-able unfortunately.

Although refactoring the options window is in my plans so I'll consider this when it's done! 😄
